src/ui/controls/ColorSwatchControl.tsx

```diff
@@ -49,25 +50,23 @@ export class ColorSwatchControl extends React.Component<IColorControlProps, void
       selectedValue,
     } = this.props.variable;
     return (
-      <div className={`${CSS.RMX_COLOR_SWATCH} ${CSS.MDL_LIST_ITEM} ${CSS.MDL_TWO_LINE}`}>
-        <span className={CSS.MDL_PRIMARY}>
-          <span>{title}
-            <span className={CSS.RMX_SELECTED_VALUE}>
-              {ColorUtils.toRgbaString(selectedValue)}
-            </span>
-          </span>
-          <span className={CSS.MDL_SECONDARY}>
-            {limitedToValues.map((value: string) => (
-              <ColorSwatch
-                color={ColorUtils.toRgbaString(value)}
-                key={value}
-                isSelected={ColorUtils.areEqual(selectedValue, value)}
-                onClick={this.onClick}
-              />
-            ))}
-          </span>
-        </span>
-      </div>
+      <ListItem
+        controlClass={CSS.RMX_COLOR_SWATCH}
+        title={title}
+        subtitle={ColorUtils.toRgbaString(selectedValue)}
+        inlineControl={false}
+        control={
+          limitedToValues.map((value: string) => (
+            <ColorSwatch
+              color={ColorUtils.toRgbaString(value)}
+              key={value}
+              isSelected={ColorUtils.areEqual(selectedValue, value)}
```

Comment:
I hope `areEqual` is a reasonably fast comparison - since it's in `render`, it will be called for every swatch on every render.

If it's at all expensive, it may be worth calculating it when the values change and caching the result.